eng/pipelines/templates/stages/archetype-sdk-tests.yml

```diff
@@ -69,6 +69,9 @@ parameters:
     type: object
     default: []
 
+variables:
```

Comment:
As we talked lets undo this and the changes to live.tests.yml because it won't work for those because we aren't using the extends for that template.